java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/kms/AwsKeyManagementServiceImpl.java

```diff
@@ -43,24 +43,25 @@
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
+/**
+ * Uses the AWS Key Management Service to provide an implementation of {@link KeyManagementService}. It provides
+ * multi-region support, i.e. you can encrypt data in one region and decrypt it using the keys from another region.
+ * The message format is:
+ *   {
+ *    "encryptedKey": "&lt;base64_encoded_bytes&gt;",
+ *    "kmsKeks": [
+ *      {
+ *        "region": "&lt;aws_region&gt;",
+ *        "arn": "&lt;arn&gt;",
+ *        "encryptedKek": "&lt;base64_encoded_bytes&gt;"
```

Comment:
i guess the html representation of angled brackets got copied.

Reply:
we cannot use `<` and `>` in the javadoc since it throws an error. Another way of doing this is `{@literal <aws_region>} but that was clunky

Reply:
ah...I faced a similar issue in C#, so I ended up with this
```    /// <code>
    /// {
    ///   "encryptedKey": base64_encoded_bytes,
    ///   "kmsKeks": [
    ///     {
    ///       "region": aws_region,
    ///       "arn": arn,
    ///       "encryptedKek": base64_encoded_bytes
    ///     },
    ///     ...
    ///   ]
    /// }
    /// </code>```

Let's decide on one and keep it consistent in both the places.